env0/resource_environment.go

```diff
@@ -67,7 +67,7 @@ func resourceEnvironment() *schema.Resource {
 			},
 			"run_plan_on_pull_requests": {
 				Type:        schema.TypeBool,
-				Description: "should run terraform plan on pull requests creations",
+				Description: "should run terraform plan on pull requests creations.\nWhen github is used, if 'true' then 'github_installation_id' must be specified as well.",
```

Comment:
Adding more details to the description seemed like the best solution.
Are there any other comments or fields I should add or add to?